### PR TITLE
add some fixtures for user traders

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: fixtures/fixtures.js [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+        "1": [
+            {
+                "id": 2,
+                "name": "Kristy Corntop",
+                "pic_url": null,
+                "market_id": 2,
+                "created_at": "2017-05-24 14:19:22",
+                "updated_at": "2017-05-24 14:19:22",
+                "deleted_at": null,
+                "trader_id": 2
```
Comment (katjam) [inline fixtures/fixtures.js]: Traders don't have trader_id

File: fixtures/fixtures.js [hunk 2]
```diff
@@ -1,6 +1,41 @@
-import Route from 'route-parser';
-
+import Store from '../src/store.js';
 var Fixtures = {
+    userTraders: {
```
Comment (katjam) [inline fixtures/fixtures.js:3]: You mean traders?

**Review by katjam — CHANGES_REQUESTED** [review]

File: fixtures/fixtures.js (hunk 2 above)
Comment (charlesstrange2) [reply fixtures/fixtures.js:3]: traders that belong to a user. ergo userTraders?

File: fixtures/fixtures.js (hunk 1 above)
Comment (charlesstrange2) [reply fixtures/fixtures.js]: aha! it's an id, yes.

File: fixtures/fixtures.js (hunk 2 above)
Comment (katjam) [reply fixtures/fixtures.js:3]: But technically speaking - according to our data - they are just trader objects. But I guess if you are just implying that this is a list of the traders attached to this user which has already been filtered by the API then maybe it makes sense.

Not going to fight you on it.

**Comment by katjam** [comment]
@charlesstrange2 Are you also going to fix these mocks - so they can be used? I think Steph has hardcoded some stuff that breaks the login flow.

**Review by katjam — APPROVED** [review]
You deserve an ice cream! :icecream: :shipit:

**Pull request merged**